Beautify overview page (#4184)
Qiskit/documentation: docs/guides/index.mdx

---
title: Overview
description: Find resources to use Qiskit, Qiskit Runtime, IBM Quantum, and more
---

{/* cspell:ignore Benchpress */}

# Overview

## Introduction to Qiskit and IBM Quantum

> **NEED TO REWRITE** The name "Qiskit" is a general term referring to a collection of software for executing programs on quantum computers. Most notably among these software tools is the open-source Qiskit SDK, and the runtime environment (accessed using Qiskit Runtime) through which you can execute workloads on IBM&reg; quantum processing units (QPUs). As quantum technology evolves, so does Qiskit, with new capabilities released every year that expand this core collection of quantum software.

Reviewer: ```suggestion
Welcome to the documentation for Qiskit, its related packages, and IBM Quantum Platform. This documentation includes how-to guides to get you started on our tools, and specific use-case tutorials that include end-to-end examples.

IBM Quantum Platform unites cutting-edge quantum hardware, advanced software tools, and a global community of researchers, developers, and industry partners working to unlock the potential of quantum computing. The open-source tools available under the Qiskit umbrella support the work of quantum algorithm researchers and quantum information scientists, and provide quantum compute resources in HPC environments. Meanwhile, IBM Quantum Platform and its proprietary services are built to enable the execution of quantum workloads through the IBM Quantum Compute Service, as well as the Qiskit Functions Catalog.
```

---
title: Overview
description: Find resources to use Qiskit, Qiskit Runtime, IBM Quantum, and more
---

{/* cspell:ignore Benchpress */}

# Overview

## Introduction to Qiskit and IBM Quantum

Welcome to the documentation for Qiskit, its related packages, and IBM Quantum Platform. This documentation includes how-to guides to get you started on our tools, and specific use-case tutorials that include end-to-end examples.

> IBM Quantum Platform unites cutting-edge quantum hardware, advanced software tools, and a global community of researchers, developers, and industry partners working to unlock the potential of quantum computing. The open-source tools available under the Qiskit umbrella support the work of quantum algorithm researchers and quantum information scientists, and provide quantum compute resources in HPC environments. Meanwhile, IBM Quantum Platform and its proprietary services are built to enable the execution of quantum workloads through the IBM Quantum Compute Service, as well as the Qiskit Functions Catalog.

Reviewer: ```suggestion
IBM Quantum Platform unites cutting-edge quantum hardware, advanced software tools, and a global community of researchers, developers, and industry partners working to unlock the potential of quantum computing. The open-source tools available under the Qiskit umbrella support the work of quantum algorithm researchers and quantum information scientists, as well as the integration of quantum compute resources in HPC environments. Meanwhile, IBM Quantum Platform and its proprietary services are built to enable the execution of quantum workloads through the IBM Quantum Compute Service, as well as the Qiskit Functions Catalog.
```

> IBM Quantum Platform unites cutting-edge quantum hardware, advanced software tools, and a global community of researchers, developers, and industry partners working to unlock the potential of quantum computing. The open-source tools available under the Qiskit umbrella support the work of quantum algorithm researchers and quantum information scientists, as well as the integration of quantum compute resources in HPC environments. Meanwhile, IBM Quantum Platform and its proprietary services are built to enable the execution of quantum workloads through the IBM Quantum Compute Service, as well as the Qiskit Functions Catalog.

Reviewer: ```suggestion
Welcome to the documentation for Qiskit, its related packages, and IBM Quantum&reg; Platform. This documentation includes how-to guides to get you started on our tools, specific use-case tutorials that include end-to-end examples, and a collection of API references.

Qiskit provides a modular and extensible framework for quantum research and development across algorithms, high-performance computing, and quantum information science. With it, researchers can build, optimize, and execute quantum workflows with specialized addons, software tools, and extensive resources. Through IBM Quantum Platform, users can access Quantum Compute services, such as Qiskit Runtime and the Qiskit Functions Catalog, to run workloads efficiently on the IBM&reg; fleet of quantum computers.
```

      linkText="Read overview"
    />

    <Card
      title="Execution"
      description="Use Qiskit Runtime primitives to run quantum circuits on IBM Quantum hardware."
      href="/docs/guides/primitives"
      analyticsName="Overview page: Execution"
      linkText="Learn more"
    />

    <Card
      title="Post-processing"
      description="Refine your results by using post-processing techniques for specialized applications available as Qiskit addons."
      href="/docs/guides/qiskit-addons-sqd"
      analyticsName="Overview page: Postprocessing"
      linkText="View SQD addon"
    />

    <Card
      title="Qiskit Functions"
      description="Accelerate quantum workload creation with pre-built tools created by partner organizations."
      href="/docs/guides/functions"
      analyticsName="Overview page: Qiskit Functions"
      linkText="Explore"
    />
</CardGroup>

## Support
<CardGroup>
    <Card
      title="Error code registry"
      description="Look up error codes and suggested solutions."
      href="/docs/errors"
      analyticsName="Overview page: Registry"
      linkText="Find an error code"
    />

    <Card
      title="Support page"

>       description="Find answers to frequently-asked questions, learn how to report bugs, discover communities to join, and more"

Reviewer: ```suggestion
      description="Find answers to frequently-asked questions, learn how to report bugs, discover communities to join, and more."
```

---
title: Overview
description: Find resources to use Qiskit, Qiskit Runtime, IBM Quantum, and more
---

{/* cspell:ignore Benchpress */}

# Overview

## Introduction to Qiskit and IBM Quantum

> **NEED TO REWRITE** The name "Qiskit" is a general term referring to a collection of software for executing programs on quantum computers. Most notably among these software tools is the open-source Qiskit SDK, and the runtime environment (accessed using Qiskit Runtime) through which you can execute workloads on IBM&reg; quantum processing units (QPUs). As quantum technology evolves, so does Qiskit, with new capabilities released every year that expand this core collection of quantum software.

Reviewer 2: Here's some copy that I think could work as a first draft, but I'd like to hear other's input on it as well before going with it.

```suggestion
Welcome to the documentation platform for Qiskit, its related packages, and the IBM Quantum Platform. Here you will find information on our collection of open-source tools as well as how to start building and running workloads on IBM Quantum hardware.

Qiskit and the IBM Quantum Platform is a place for exploring, developing, and deploying quantum computing algorithms and workloads. It brings together our cutting-edge quantum hardware, advanced software tools, and a global community of researchers, developers, and industry partners working to unlock the potential of quantum computing. At the heart of this platform is Qiskit—a comprehensive collection of open-source tools designed to help users build and run quantum programs across a variety of quantum systems, all maintained by the community.

Meanwhile the IBM Quantum Platform, and its runtime environment (accessed using Qiskit Runtime) is the place through which you can execute workloads on IBM® quantum processing units (QPUs). As quantum technology evolves, so does our software, with new capabilities released every year that expand this core collection of quantum tools.
```

Reviewer: Nice! Remembering that we have much of this content in the actual introduction page to Qiskit, which we are not deleting, maybe we can cut this down more? Here are my proposed edits:

> Welcome to the documentation for Qiskit, its related packages, and IBM Quantum Platform. This documentation includes how-to guides to get you started on our tools, and specific use-case tutorials that include end-to-end examples.

> IBM Quantum Platform unites cutting-edge quantum hardware, advanced software tools, and a global community of researchers, developers, and industry partners working to unlock the potential of quantum computing. At the heart of this platform is Qiskit - a comprehensive collection of open-source tools for building and running quantum programs. Users can simulate programs locally, or can use Qiskit Runtime to execute workloads on IBM quantum processing units (QPUs) in the cloud.

Reviewer 2: This LGTM!

Reviewer: (We're doing another iteration of crafting this in order to reflect the image we now have)

Reviewer 2: Here's some new copy for this based on feedback:

>  Welcome to the documentation for Qiskit, its related packages, and IBM Quantum Platform. This documentation includes how-to guides to get you started on our tools, and specific use-case tutorials that include end-to-end examples.

> The IBM Quantum Platform unites cutting-edge quantum hardware, advanced software tools, and a global community of researchers, developers, and industry partners working to unlock the potential of quantum computing. The suite of open-source tools available under the "Qiskit" umbrella are designed to enable research for quantum algorithms, quantum information science, as well as utilizing quantum compute resources in HPC environments. Meanwhile the IBM Quantum Platform and its proprietary services are built to enable the execution of quantum workloads through the IBM Quantum Compute Service as well as the Qiskit Functions Catalog.

![TEXT FOR RAINBOW IMAGE](/docs/images/qiskit-patterns/patterns.svg)

> While not a part of Qiskit itself, the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) is a collection of open-source projects that interface with Qiskit and can add valuable functionality.

Reviewer 2: ```suggestion
While not a part of Qiskit itself, we also maintain the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) -- a collection of open-source projects that interface with Qiskit which can add valuable functionality.
```

Reviewer: Possible tweaks:

> Beyond Qiskit and its related packages is the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem), a catalog of open-source projects that interface with Qiskit to extend its functionality.
